src/test/java/step3/LottoTest.java

```diff
@@ -0,0 +1,26 @@
+package step3;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import step3.domain.Lotto;
+import step3.domain.LottoNumbers;
+
+public class LottoTest {
+    @Test
+    @DisplayName("로또 번호 비교 로직 테스트")
+    public void compareTest() {
+        LottoNumbers source = LottoNumbers.ofManual(
+            List.of(1, 2, 3, 4, 5, 6)
+        );
+        LottoNumbers target = LottoNumbers.ofManual(
+            List.of(1, 2, 3, 4, 5, 7)
+        );
+        Lotto sourceLotto = new Lotto(source);
+        assertTrue(sourceLotto.match(target, 8).verifyMatchCount(5));
```

Comment:
```suggestion
        MatchResult actual = sourceLotto.match(target, 8);

        assertThat(actual).isEqualTo(new MatchResult(5, false));
```

객체간 비교는 어떤가요?

---

`verifyMatchCount`는 별도로 `MatchResultTest`에 있는게 적합해보이네요 🤔 
```java

public class MatchResultTest {

...
    @Test
    void verifyMatchCount_test() {
        MatchResult matchResult = new MatchResult(5, true);

        assertThat(matchResult.verifyMatchCount(5)).isTrue();
    }
```